src/main/webapp/src/app/hidden-gem.service.ts

```diff
@@ -17,8 +17,10 @@ export class HiddenGemService {
       .pipe(catchError(this.handleError))
   }
```

Comment:
Is the intention to remove this eventually? or does it still have a use?
maybe add a comment either with a todo to remove or maybe something say it's staying around for debugging purposes for now?